Makefile

```diff
@@ -257,6 +257,10 @@ manifests: generate-manifests $(KUSTOMIZE)
 unit: manifests
 	go test ./api/... ./controllers/... ./baremetal/... -coverprofile cover.out
 
+unit-cover-html: manifests
+	go test ./api/... ./controllers/... ./baremetal/... -coverprofile cover.out
```

Comment:
you could remove this line and add a dependency on unit instead

Reply:
done